test/boost/tablets_test.cc

```diff
@@ -1738,3 +1740,120 @@ SEASTAR_THREAD_TEST_CASE(test_load_balancing_resize_requests) {
         }
     }).get();
 }
+
+static locator::endpoint_dc_rack make_endpoint_dc_rack(gms::inet_address endpoint) {
+    // This resembles rack_inferring_snitch dc/rack generation which is
+    // still in use by this test via token_metadata internals
+    auto dc = std::to_string(uint8_t(endpoint.bytes()[1]));
+    auto rack = std::to_string(uint8_t(endpoint.bytes()[2]));
+    return locator::endpoint_dc_rack{dc, rack};
+}
+
+SEASTAR_THREAD_TEST_CASE(test_tablet_reallocation_kq) {
```

Comment:
the name has to change, especially the `kq` ending ;) 
and again, we're not "reallocating" any tablet replica, either removing them or adding new ones. Rellocation reads as if we were moving them somewhere, e.g. between nodes/shards, which is not the case.

Reply:
Oops, I totally missed that.